### PR TITLE
"ABOUT" alignment fix

### PR DESCRIPTION
**Pull request by jatin0814**
Screenshot (18)

### CODE REVIEW

**Comment by jatin0814** [comment]
@prathmeshkhandelwal1  check kar le

File: src/Component/HomeComponents/About/About.css [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+  display: flex;
+  justify-content: space-between;
+  
+}
+  .ABOUT {
     width: 4.2em;
-    margin: 0em 2.2em;
+    
     padding: 1em;
     border-bottom: 7px solid #821169;
```
Comment (prathmeshkhandelwal1) [inline src/Component/HomeComponents/About/About.css]: make this width 3px

File: src/Component/HomeComponents/About/About.css [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
-.ABOUT {
+.About_root{
+  margin: 15% 5%;
+}
+.About_main{
+  display: flex;
+  justify-content: space-between;
+  
+}
+  .ABOUT {
```
Comment (prathmeshkhandelwal1) [inline src/Component/HomeComponents/About/About.css:9]: width and height is large as compared to gallery header.

**Review by prathmeshkhandelwal1 — CHANGES_REQUESTED** [review]
@jatin0814  make about heading small and also the size of box is bit large.

File: src/Component/HomeComponents/About/About.css (hunk 1 above)
Comment (prathmeshkhandelwal1) [reply src/Component/HomeComponents/About/About.css]: > make this width 3px

border-bottom property

**Comment by jatin0814** [comment]
Screenshot (20)
@prathmeshkhandelwal1 now?

**Review by prathmeshkhandelwal1 — APPROVED** [review]

**Pull request merged**